ledger/appcow.go

```diff
@@ -619,7 +638,7 @@ func applyStorageDelta(data basics.AccountData, aapp storagePtr, store *storageD
 			delete(owned, aapp.aidx)
 		case allocAction, remainAllocAction:
 			// note: these should always exist because they were
-			// at least preceded by a call to PutWithCreatable
+			// at least preceded by a call to Put()
```

Comment:
```
at least preceded by a call to AllocateApp
```

Reply:
This is looking up in `owned` which is a copy of `data.AppParams`. It is set in `Put()`, isn't it?

Reply:
This is correct, the comment most likely refers to `owned` rather than `store.action`

Reply:
so there is no problem?

Reply:
yes, all looks correct